tests/modules/haplogrep2/classify/main.nf

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { HAPLOGREP2_CLASSIFY } from '../../../../modules/haplogrep2/classify/main.nf'
+
+workflow test_haplogrep2_classify {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['homo_sapiens']['illumina']['test_mito_vcf'], checkIfExists: true)
+    ]
+    format = 'vcf'
+
+    HAPLOGREP2_CLASSIFY ( input,format )
+}
+
+workflow test_haplogrep2_classify_stub {
```

Comment:
We probably don't need this workflow for stub run, the `-stub-run` parameter at the _haplogrep2 classify stubs_ test command should be enough.

Then the `--entry` would be the same workflow name as the test `haplogrep2 classify`: _test_haplogrep2_classify_